src/js/os.js

```
@@ -16,6 +16,42 @@ exports.freemem = function() {
   return native.getFreeMem();
 };
 
+exports.getPriority = function(pid) {
+  if (pid === undefined) {
+    pid = 0;
+  } else {
+    pid = Number(pid);
+  }
+
+  var err = native.getPriority(pid);
+  if (err instanceof Error) {
+    throw err;
+  }
+  return err;
+};
+
+exports.setPriority = function(pid, priority) {
+  if (priority === undefined) {
+    priority = pid;
+    pid = 0;
+  } else {
+    pid = Number(pid);
+  }
+
+  if (typeof priority !== 'number') {
+    throw new TypeError('priority must be a number');
+  }
+  if (priority > 19) {
+    priority = 19;
+  } else if (priority < -20) {
+    priority = -20;
```

Should an error been thrown instead of silently rounded?